host/src/layouts/Auth.layout.tsx

```diff
@@ -1,27 +1,63 @@
-import { Link, Outlet } from "react-router-dom";
+import { Link, Outlet, useLocation } from "react-router-dom";
+import HelmetHeader from "@/components/HelmetHeader";
 import { HomeHashtag } from "iconsax-react";
 import { AuthRoutes } from "@/types/routes";
 
+type AuthRoutesPath = (typeof AuthRoutes)[keyof typeof AuthRoutes];
```

Comment:
You're duplicating the type `AuthRoutes` here. Use `AuthRoutes` the way you intend to use `AuthRoutesPath`

```suggestion

```